domain_specific.tex

```diff
@@ -4,8 +4,6 @@
 mechanics along with support for manipuating physical quantities with units.
 
 
-\subsection{Classical Mechanics}
```

Comment:
This should not have been removed, the below levels are sub-sub-section, this is the relevant sub-section